test-monitor.cpp

```diff
@@ -1,21 +1,57 @@
+#include <memory>
 #include "./monitor.h"
 #include "gtest/gtest.h"
+#include "./localization.h"
 
-TEST(Monitor, IndividualVitalsCheck) {
+// Temperature ranges = {95,96.53,100.47,102}

```

Comment:
Instead of manually adjusting the numbers, you can switch to property-based tests.
In this case, for example, you could move constants like TEMP_LIMIT_HIGH, TEMP_TOLERANCE_PERCENT, etc to the header file and use it here... like TEMP_LIMIT_HIGH+1 instead of 103.5
This can assert the "property" that "one-degree above high limit is hyperthermia"

Reply:
Sure. Will do that.